packages/sdk/src/node.ts

```diff
@@ -165,3 +184,47 @@ export function loadScriptNode(
     );
   });
 }
+
+async function loadModule(
+  url: string,
+  options: {
+    vm: any;
+    fetch: any;
+  },
+  parentContext?: any,
+) {
+  const { fetch, vm } = options;
+  const context =
+    parentContext ||
+    vm.createContext({
+      ...global,
+      Event,
+      URL,
+      URLSearchParams,
+      TextDecoder,
+      TextEncoder,
+      console,
+      require: eval('require'),
+      __dirname,
```

Comment:
bundler will transform these into strings in other targets, like browser etc. cant use short hand
```suggestion
      __dirname: __dirname,
      __filename: __filename,
```